yt_dlp/extractor/clubhouse.py

```diff
@@ -0,0 +1,30 @@
+from .common import InfoExtractor
+
+
+class ClubhouseIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?clubhouse\.com/room/(?P<id>[a-z0-9A-Z]+)'
+
+    _TESTS = [{
+        'url': 'https://www.clubhouse.com/room/PN50O4GG',
+        'info_dict': {
+            'id': 'PN50O4GG',
+            'ext': 'ts',
+            'title': '香港消費券轉會！八達通、PayMe、BoC Pay、Tap＆Go、AlipayHK、WeChat Pay 邊個最有伏'
+        }
+    }]
+
+    def _real_extract(self, url):
+        id = self._match_id(url)
```

Comment:
I think this field should be renamed to something else since there's `id` in builtins :sweat_smile:

Reply:
I haven't been following core changes for past 6+ months...are there any major changes?
Bcoz i've always used `id` in all my extractors 🤔

Reply:
nothing stops from using `id` variable at least

Reply:
Generally it's bad to shadow builtins. However the `id` builtin is rarely used. So I generally don't bother correcting this.

(I personally prefer to use `item_id`/`video_id`/`display_id` though)

Reply:
if it is not a breaking change then I'd rather keep it id only. The key in return dict is `id`, so it makes way more sense to call the var `id` from my perspective. 

I use item_id or video_id in playlist extractors and display_id is a different key all together so it doesn't make sene to me to use those.

Marking this thread resolved.